clients/instance/ibm-pi-instance.go

```diff
@@ -253,18 +253,18 @@ func (f *IBMPIInstanceClient) GetSnapShotVM(id string) (*models.Snapshots, error
 }
 
 // Restore a Snapshot of an Instance
-func (f *IBMPIInstanceClient) RestoreSnapShotVM(id, snapshotid, restoreAction string, body *models.SnapshotRestore) (*models.Snapshot, error) {
+func (f *IBMPIInstanceClient) RestoreSnapShotVM(instanceID, snapshotID, restoreFailAction string, body *models.SnapshotRestore) (*models.Snapshot, error) {
```

Comment:
IMO appending `instance` to the ID seems redundant. The functions in InstanceClient as particularly related to Instances.
Eg: In Volume client,`id` should mean volumeID and for instance, instanceID (operations like Attach/Detach).

At many places, it is named as pvmInstanceID or pvmID or something else.